src/main/java/com/promise8/wwwbe/service/MeetingService.java

```diff
@@ -87,6 +90,21 @@ public MeetingCreateResDto createMeeting(MeetingCreateReqDto meetingCreateReqDto
         return MeetingCreateResDto.of(meetingCode, getDynamicLink(meetingCreateReqDto.getPlatformType()));
     }
 
+    @Transactional
+    public void putMeetingStatus(long meetingId, ActionType actionType) {
+        MeetingEntity meetingEntity = meetingRepository.findById(meetingId).orElseThrow(() -> {
+            throw new BizException(BaseErrorCode.INVALID_REQUEST, "not exist meeting");
+        });
+
+        if (ActionType.END_VOTE.equals(actionType)) {
+            meetingEntity.setMeetingStatus(MeetingStatus.VOTED);
```

Comment:
Service에서 setter 사용하는것 보다는 entity에 메소드 만드는게 더 좋아보여요!

Reply:
옙 다음번 커밋올릴때 수정하겠습니당~

Reply:
@emost22 
근데 요거 생각해보니까, entity에서 메소드를 만든다하더라도 setter랑 동일시되는 다른 네이밍의 동일한기능의 메소드일듯한데 (MeetingStatus를 변경시키는) 무의미하지않을까?
```suggestion
            meetingEntity.changeMeetingStatus(MeetingStatus.VOTED);
```

아니면 ActionType을 meetingEntity 내부에서 활용한다고 생각한것인지!? (그 경우는 지난번에 Entity에 DTO레벨 객체는 포함시키지 않는것으로 정책을 세웠던 것으로 기억해서)
Entity 객체 내부의 값들을 활용하는 public method를 만드는건 괜찮은거같은데 이 경우는 위에처럼 가는게 낫다는 생각이 드네유

Reply:
setter를 사용하지 말자는게 객체 생성시점 상황을 생각했었는데(쓰레드세잎 안정성 측면?.. 지금은 고려할일도없지만!)
setter자체를 사용안하고 다른 메소드로 이름을 포장해서 쓰는건 무의미한것 같아용!